packages/wonder-blocks-form/src/components/labeled-text-field.stories.js

```diff
@@ -121,6 +164,36 @@ export const Email: StoryComponentType = () => {
     );
 };
 
+export const EmailRequired: StoryComponentType = () => {
```

Comment:
Same comment related to chromatic applies with this story: `disableSnapshot: true`